tools/tasks/seed/build.js.prod.ts

```diff
@@ -1,8 +1,9 @@
 import * as gulp from 'gulp';
 import * as gulpLoadPlugins from 'gulp-load-plugins';
 import { join } from 'path';
+import * as merge from 'merge-stream';
 
-import { TMP_DIR, TOOLS_DIR } from '../../config';
+import { TMP_DIR, TOOLS_DIR, INLINE_TEMPLATES, BOOTSTRAP_DIR, PROD_DEST } from '../../config';
```

Comment:
It'd be great if you could update the imports to conform to the styleguide :)

https://angular.io/styleguide#!#coding-conventions
> Import Line Spacing
> Style 03-06
> Do list destructured imported assets alphabetically.